core/conn/unixodbc/odbc/odbcclient/unixcli/cli/ctosqlconv.cpp

```diff
@@ -3545,7 +3545,7 @@ unsigned long ODBC::ConvertCharToNumeric(SQLPOINTER srcDataPtr,
     rTrim(cTmpBuf);
     tempLen = strlen(cTmpBuf);
     errno = 0;
-	dTmp = strtod(cTmpBuf, &errorCharPtr);
+	dTmp = strtold(cTmpBuf, &errorCharPtr);
```

Comment:
This looks innocuous, but strlod return long double which is 16 byte.  This 16 byte is assigned to 8 byte double dTmp. I am not sure if there is an implicit casting to double from long double can work as expected.

Reply:
Yeah, when I tried it I found on Red Hat Linux long double was 16 bytes. But on Windows 8, it is just 8 bytes. It depends on the platform. For what it is worth, I found that the implicit cast from long double to double worked fine on both platforms.

Reply:
Thanks Dave. In that case, this change looks good to me